taiga Bot/dist/metadata.json

```diff
@@ -0,0 +1,35 @@
+{
+    "author": {
+        "name": "CRUGG",
+        "id": "228965621478588416"
+    },
+    "service": "taiga Bot",
+    "description": {
+        "en": "taiga is a Discord Bot with many features like Virtual Trading Cards, an economy system and in the future even a Plugin Marketplace",
+        "de": "taiga ist ein Discord Bot mit vielen Funktionen wie virtuellen Sammelkarten, ein Wirtschaftssystem und in der Zukunft ein Pluginmarktplatz"
+    },
+    "url": [
+        "taigabot.net",
+        "anigifs.taigabot.net",
+        "api.taigabot.net",
+        "beta.taigabot.net",
+        "cdn.taigabot.net",
+        "cdnadmin.taigabot.net",
+        "community.taigabot.net",
+        "git.taigabot.net",
+        "labs.taigabot.net",
+        "legal.taigabot.net",
+        "taipl.taigabot.net",
+        "telegram.taigabot.net",
+        "translate.taigabot.net"
+    ],
+    "version": "1.0",
+    "logo": "https://cdn.taigabot.net/assets/branding/taiga/premid_logo.png",
+    "thumbnail": "https://cdn.taigabot.net/assets/branding/taiga/premid_banner.png",

```

Comment:
upload to imgur.

Reply:
There is no presence guideline which says that "its not allowed to be on the cdn of the site" like you told me on Discord. Also there is like literally no difference between me hosting images on my private site and me hosting images on my project's site. They both run on the same VPS and even use the same Webserver Instance.

Reply:
Imgur retains images forever, disregarding the hiccups here and there. But there's no guarantee that the CDN will keep the name of the logo the same. For example, it could be changed to `icon.png` on the cdn and will break the presence's icon.

Reply:
You did use your private CDN on another presence so it'll be fine to use this one.

Reply:
@Alanexei The only way the icon name could change would be that I use an SFTP Client, login via SSH or use our Admin Interface and manually change it.